File: OCH_Tao/ABC/B/146.js
const X = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
const [N, S] = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"));
const Y = X.slice(parseInt(N, 10)) + X.slice(0, parseInt(N, 10));
let L = [];
for (let i = 0; i < S.length; i++) {
  L.push(Y[X.indexOf(S[i])]);
}
console.log(L.join(""));

File: OCH_Tao/ABC/B/146.py
X = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
N = int(input())
S = input()
Y = X[N:]+X[:N]
L = []
for i in range(len(S)):
  L.append(Y[X.index(S[i])])
print("".join(L))